certbot/compat/filesystem.py

```diff
@@ -173,3 +235,14 @@ def _compare_dacls(dacl1, dacl2):
     """
     return ([dacl1.GetAce(index) for index in range(0, dacl1.GetAceCount())] ==
             [dacl2.GetAce(index) for index in range(0, dacl2.GetAceCount())])
+
+
+def _get_current_user():
+    """
+    Return the pySID corresponding to the current user.
+    """
+    account_name = win32api.GetUserNameEx(win32api.NameSamCompatible)
+    # Passing None to systemName instruct the lookup to start from the local system,
+    # then continue the lookup to associated domain.
```

Comment:
what does this mean? continues the lookup...where? how? what is looking up what domain? I suspect this is a perfectly reasonable choice and it's just the language here.

Reply:
Indeed, my comment was not clear. I improved it.